src/redux/features/appVersion/appVersionSlice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+// Define a type for the slice state
+interface AppVersionState {
+  version?: string;
+}
+
+// Define the initial state using that type
+const initialState: AppVersionState = {};
+
+export const appVersionSlice = createSlice({
```

Comment:
here is a simple reducer created via createSlice that at this time just allows us to set a simple version state